documentation/pages/move-registry/managing-package-info.mdx

```diff
@@ -1,13 +1,120 @@
 import Beta from '../../components/beta'
+import { Callout } from 'nextra/components';
 
 <Beta/>
 
 # Creating/Maintaining `PackageInfo` object
 
-[
-TODO!!!: Add sui client ptb commands to:
-1. Create a `PackageInfo` object for your package.
-2. Add the source code information to the `PackageInfo` object.
+<Callout type="warning">
+    These examples assume that you use the Typescript Plugin, as described in the  [`Typescript SDK`](/move-registry/tooling/typescript-sdk) section.
```

Comment:
```suggestion
    These examples assume that you use the MVR plugin for the Sui Typescript SDK, as described in the  [`Typescript SDK`](/move-registry/tooling/typescript-sdk) section.
```
I think wording like this makes it a little clearer what it is? I think it's technically correct?